bookwyrm/templates/preferences/edit_user.html

```diff
@@ -61,6 +61,10 @@
         {% url 'directory' as path %}
         <p class="help">{% blocktrans %}Your account will show up in the <a href="{{ path }}">directory</a>, and may be recommended to other BookWyrm users.{% endblocktrans %}</p>
     </div>
+    <div>
+        <label class="dropdown label" for="id_preferred_timezone">{% trans "Preferred Timezone: " %}</label>
+        {{ form.preferred_timezone }}
+    </div>
```

Comment:
for stylistic consistently, this will work best if the `select` element itself on line 66 is wrapped in a div with class `select` (https://bulma.io/documentation/form/select/), and the div on line 54 gets the class `block`. I believe `dropdown` is intended for html menus, not for select form elements.

It's fussy, sorry! I can also fix it later if you prefer not to get into the weeds on markup.

Reply:
Front end is my weak point. Thanks for this note. I can read up on this and try to address it or I'll ask you for help.

Reply:
it's one of those things that's easy if you already know the css framework, and super non-obvious when you don't

Reply:
Looks better!

![image](https://user-images.githubusercontent.com/454944/112775944-49f91500-900c-11eb-9c0a-effbe608c1ec.png)